### PR TITLE
clarify ado behavior

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr clarifies that for ado, private packages have to be configured using the `org/repo` path. 

more details in internal slack: https://dbt-labs.slack.com/archives/C07C793M6EB/p1734715084356599?thread_ts=1734359013.848149&cid=C07C793M6EB

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-mirnawong1-patch-28-dbt-labs.vercel.app/docs/build/packages

### CODE REVIEW

File: website/docs/docs/build/packages.md [hunk 1]
```diff
@@ -173,19 +173,36 @@ To use native private packages, you must have one of the following Git providers
 
 #### Configuration
 
-Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
+Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable. 
+
+The private key supports a two path layer: `org/repo` and dbt inherites the project path from your source repository.
```
Comment (mirnawong1) [inline website/docs/docs/build/packages.md]:
```suggestion
The private key supports a two-layered-path: `org/repo`. dbt inherits the project path from your source repository.
```

File: website/docs/docs/build/packages.md [hunk 2]
```diff
@@ -173,19 +173,36 @@ To use native private packages, you must have one of the following Git providers
...
 packages:
-  - private: dbt-labs/awesome_repo
+  - private: dbt-labs/awesome_repo # your-org/your-repo path
   - package: normal packages
-
-	[...]
+  [...]
 ```
 
+:::tip
```
Comment (mirnawong1) [inline website/docs/docs/build/packages.md]: maybe this shouldn't be a callout - can make this expandable or h4?

File: website/docs/docs/build/packages.md [hunk 3]
```diff
@@ -173,19 +173,36 @@ To use native private packages, you must have one of the following Git providers
...
+  - private: my-org/my-repo # Works if your source repo and package repo are in the same project
+```
 </File>
+:::
+
+For Azure DevOps repositories, you can only specify the `org/repo` path. The project tier is inherited from the source repository currently integrated with dbt Cloud.
+
+- If the package repository is within the same project as your source repository, you’re in the clear. Simply specify org/repo in the private block.
+- If the package repository is not within the same project as your source repository, this feature will not work for you until a future update.
+- This currently only works within a single Azure DevOps project. If your repositories are in different projects within the same organization, you can't reference them in the `private` key at this time.
```
Comment (ericbaumann) [inline website/docs/docs/build/packages.md]: Could we add this as a "Prerequisite" of the feature? That, for ADO, the repo is located within the same org/project as your project's repo.

**Review by ericbaumann — COMMENTED** [review]
Thanks for the quick turnaround on updating these! 

i'd recommend making sure that the bits about org/project/repo are indicated as an ADO concept

File: website/docs/docs/build/packages.md (hunk 3 above)
Comment (runleonarun) [reply website/docs/docs/build/packages.md]: Agree with @ericbaumann here! We should move these bullets up to the "Prereqs" section.

File: website/docs/docs/build/packages.md [hunk 4]
```diff
@@ -173,19 +173,37 @@ To use native private packages, you must have one of the following Git providers
...
-  - private: dbt-labs/awesome_repo
+  - private: dbt-labs/awesome_repo # your-org/your-repo path
   - package: normal packages
-
-	[...]
+  [...]
 ```
+</File>
+
+:::tip Azure DevOps repositories
```
Comment (runleonarun) [inline website/docs/docs/build/packages.md]: Maybe this should be titled "Azure DevOps limitations?"

File: website/docs/docs/build/packages.md [hunk 5]
```diff
@@ -173,19 +173,37 @@ To use native private packages, you must have one of the following Git providers
...
+Normally ADO repos use the `org_name/project_name/repo_name` path. However, private package supports only the `org/repo` path. If the ADO package repository is within the same project as your source repository, you can specify `org/repo` in the `private` key. If the package repository _isn't_ within the same project as your source repository, then the private packages feature won't currently work:
```
Comment (runleonarun) [inline website/docs/docs/build/packages.md]: I recommend starting with the take away from  this callout. Maybe we can also simplify it a little?

```suggestion
Currently, the private package feature only works when the ADO package repository is within the same project as your source repository. Normally ADO repos use the `org_name/project_name/repo_name` path. However, private package supports only the `org/repo` path. 

You can use private packages by specifying `org/repo` in the `private` key:
```

**Review by runleonarun — COMMENTED** [review]
Thanks @mirnawong1 This is great and I appreciate you spinning this PR up!

Added a few suggestions.

File: website/docs/docs/build/packages.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/build/packages.md]: that's a good flag! i usually try use more positive language and tend to use considerations more than limitations. would Azure DevOps considerations work?

Comment (mirnawong1) [reply website/docs/docs/build/packages.md]: Screenshot 2024-12-24 at 10 04 24

File: website/docs/docs/build/packages.md (hunk 3 above)
Comment (mirnawong1) [reply website/docs/docs/build/packages.md]: thanks @ericbaumann @runleonarun - updated it to this:

Screenshot 2024-12-24 at 10 04 21

File: website/docs/docs/build/packages.md [hunk 6]
```diff
@@ -165,27 +165,45 @@ dbt Cloud supports private packages from [supported](#prerequisites) Git repos l
...
 
-To use native private packages, you must have one of the following Git providers configured in the **Integrations** section of your **Account settings**:
-- [GitHub](/docs/cloud/git/connect-github)
-- [Azure DevOps](/docs/cloud/git/connect-azure-devops)
-- Support for GitLab is coming soon.
-
+- To use native private packages, you must have one of the following Git providers configured in the **Integrations** section of your **Account settings**:
+  - [GitHub](/docs/cloud/git/connect-github)
+  - [Azure DevOps](/docs/cloud/git/connect-azure-devops)
+    - For Azure DevOps, use the `org/repo` path (not the `org_name/project_name/repo_name` path) with the project tier inherited from the integrated source repository.
```
Comment (ericbaumann) [inline website/docs/docs/build/packages.md]: I think the important thing to emphasize as a pre-req is that they only work in a single project within ADO. The specific syntax, I think you do a good job of covering below.

File: website/docs/docs/build/packages.md [hunk 7]
```diff
@@ -165,27 +165,45 @@ dbt Cloud supports private packages from [supported](#prerequisites) Git repos l
...
+    - Private packages only work within a single Azure DevOps project. If your repositories are in different projects within the same organization, you can't reference them in the `private` key at this time.
+  - Support for GitLab is coming soon.
 
 #### Configuration
 
-Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
+Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable. 
+
+The private key supports a two-layered-path: `org/repo`. dbt inherits the project path from your source repository.
```
Comment (ericbaumann) [inline website/docs/docs/build/packages.md]: I would just leave this out - the project is kind of a concept specific to ADO, so it's not exactly correct to say that that's the behavior for other providers.

**Review by ericbaumann — APPROVED** [review]

File: website/docs/docs/build/packages.md (hunk 7 above)
Comment (mirnawong1) [inline website/docs/docs/build/packages.md]:
```suggestion
```

**Review by runleonarun — APPROVED** [review]
Looks great! Thanks @mirnawong1!  Merging this too.

**Pull request merged**